src/createServer.js

```diff
@@ -1,3 +1,66 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/index');
+
+const caseOptions = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const [text, toCase] = req.url.substring(1).split('?toCase=');
```

Comment:
Cool idea to split by `?toCase=` :fire: 
```suggestion
    const [text, toCase] = req.url.substring(1).split('?toCase=');
```